tests/installation/partitioning_lvm.pm

```diff
@@ -75,6 +87,48 @@ sub run {
             $collect_logs = 1;
         }
     }
+    elsif (get_var("STORAGE_NG")) {
+        if ($numdisks <= 1) {
+            die "Guided workflow does not skip disk selection when only one disk!"
+                if !match_has_tag('inst-partitioning-scheme');
+        }
+        else {
+            die "Guided workflow does skip disk selection when more than one disk! (NUMDISKS=$numdisks)"
+                if !match_has_tag('inst-partition-guided');
+            assert_screen "inst-partition-guided";       
+            send_key 'alt-n';
+            assert_screen "inst-select-root-disk";
+            send_key 'alt-n';
+        }
+        send_key 'alt-e';
+        assert_screen "inst-partitioning-lvm-enabled";
+        if (get_var("ENCRYPT")) {
+            send_key 'alt-a';
+
+            if (!get_var('ENCRYPT_ACTIVATE_EXISTING')) {
+                assert_screen 'inst-encrypt-password-prompt';
+                #send_key 'tab'; # might be needed again later 
+                type_password;
+                send_key 'tab';
+                type_password;
```

Comment:
Here is a little tricky part, as what we have seen sometimes that we press keys to fast and for example start typing password before we got into password field. But let's see if we experience this issue.